plugins/backend/rag-ai-backend-embeddings-aws/package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "@roadiehq/rag-ai-backend-embeddings-aws",
   "description": "The AWS (Bedrock) backend module for the @roadiehq/rag-ai plugin.",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
No need to bump these manually, the changeset and release based on that will do it.

Reply:
Should i revert all versions ?

Reply:
yup, better leave all versions as is. If the package is mentioned in the changeset, it will be bumped automatically